src/crypto.js

```diff
@@ -1,10 +1,22 @@
 import { ERRORS, checkObject, MasqError } from './errors'
 
+const _checkEncodingFormat = (format) => {
+  if (format !== 'hex' && format !== 'base64') throw new MasqError(ERRORS.INVALID_ENCODING_FORMAT)
+}
+
 const genRandomBuffer = (len = 16) => {
   const values = window.crypto.getRandomValues(new Uint8Array(len))
   return Buffer.from(values)
 }
 
+const genRandomBufferAsStr = (len = 16, encodingFormat) => {
+  if (encodingFormat) {
+    _checkEncodingFormat(encodingFormat)
+  }
+  const values = window.crypto.getRandomValues(new Uint8Array(len))
```

Comment:
you should reuse your function `genRandomBuffer` here :)

Reply:
yes, right.